.travis.yml

```diff
@@ -4,7 +4,9 @@ branches:
   only:
     - master
 script:
-  - "RUBYOPT='-w' bundle exec rake $TASK"
+  - bundle exec rake $TASK
+env:
+  - RUBYOPT='-w --enable-frozen-string-literal'
```

Comment:
I'm fine with introducing `frozen_string_literal: true` and supporting `--enable-frozen-string-literal` with best-effort basis, however, I don't think it's a good idea to run CI with the non-default behavior. Besides, as long as we specify `frozen_string_literal: true` in every file, the behavior would be sufficiently tested anyway, at least inside Hamlit.

Reply:
Though, I'd be okay with adding the following config:

```yml
    - rvm: 2.7.0
      env: TASK=test RUBYOPT='-w --enable-frozen-string-literal'
```
in the `matrix.include` if you like.

Reply:
The problem I see with `frozen_string_literal: true` is that in the long term the files will get moved around and new files will be introduced, and the comment might get forgotten.

I'd be happy with an extra job which runs the suite with `--enable-frozen-string-literal` option.